app/src/main/assets/native/externalplayer.js

```diff
@@ -29,7 +29,7 @@ define(['events', 'playbackManager', 'toast'], function (events, playbackManager
         };
 
         self.canPlayItem = function (item, playOptions) {
-            var mediaSource = item.MediaSources[0] || false;
+            var mediaSource = item.MediaSources && item.MediaSources[0] || false;
```

Comment:
Basically the `&&` will convert both the left and right side to booleans. The result will be `true` or `false.
This will then end up doing `true || false`. So the `|| false` part can be removed.

Reply:
Fixed. Easy commit farming I guess :D